extensions/mssql/src/objectExplorerNodeProvider/hdfsCommands.ts

```diff
@@ -168,6 +168,7 @@ export class MkDirCommand extends ProgressCommand {
 	}
 
 	private async mkDir(fileName, folderNode: FolderNode, cancelToken: vscode.CancellationTokenSource): Promise<void> {
+		await folderNode.mkdir(fileName);
```

Comment:
This function seems like unnecessary abstraction since its a 1 line function anyway. Couldn't the caller just directly call mkdir?